app/src/main/java/org/mozilla/tv/firefox/session/SessionRepo.kt

```diff
@@ -49,4 +50,11 @@ class SessionRepo(private val sessionManager: SessionManager, private val sessio
     fun setDesktopMode(active: Boolean) = sessionManager.selectedSession?.let { it.desktopMode = active }
 
     fun reload() = sessionUseCases.reload.invoke()
+
+    fun exitFullScreenIfPossibleAndBack() {
```

Comment:
This feels a little like a UI interaction, but I guess "fullscreen" is kind of a state as well.

Reply:
That's a good point.  Where is the proper place to manage "fullscreen" staus?  I'm honestly not sure.